### DISCUSSION TITLE
Upload config applied to non-upload requests

### DISCUSSION THREAD

**Discussion by khakimvinh** [payloadcms/payload#2274] [Feature Requests & Ideas]
# Bug Report

When defining `upload` config in `payload.config.ts`, the config limit will also be applied to non-upload requests.

## Steps to Reproduce

1. Add in `upload` config to `payload.config.ts`, for example:
```ts
  upload: {
    limits: {
      fieldSize: 256, // 256 bytes
    },
  },
```
2. Restart servers, now any request like, creating docs which bigger than 256 bytes will hit below error
```bash
ERROR (payload): SyntaxError: Unterminated string in JSON at position 256
```
## Other Details

Tested on version: 1.6.17
Cloned `payload-demo` and reproduce the error: https://github.com/khakimvinh/payload-upload-bug.git

**Comment by jmikrut** [comment]
Hey @khakimvinh — this is actually working as intended, and I think the confusion stems from the way that the top-level config property is named. This limit is used within the `express-fileupload` middleware, which is applied at the top level. 

Is this preventing you from achieving something that you need? We could try and scope this middleware only to the endpoints that accept uploads (collection create / update) but I believe there were some additional complexities in that which is why we kept the `express-fileupload` middleware scoped to the top-level `/api` router.

I will convert this to a discussion since it's technically as intended, although I will tag it as a feature request. We'd love to know if you need this functionality - - and why, and from there, we can prioritize the change.

Let us know!

**Comment by khakimvinh** [comment]
Hi @jmikrut, thanks for the input. 
My confusion is this limits is set under the `upload` part of `payload.config.ts`, shouldn't it only applies to any upload related request?

For example, right after cloning the `public-demo`, adding in this into `payload.config.ts`:
```ts
...
  upload: {
    limits: {
      fieldSize: 128,
    },
  },
...
```
Then even create `category` with long title will be limited, this should not have anything to do with file-upload 
![image](https://user-images.githubusercontent.com/24789737/223877729-24c8bc70-f086-40ee-814b-7ee60729bd76.png)
